### PR TITLE
modify readme (re-submit pr2142)

### PR DESCRIPTION
**Pull request by whitewum**
### Checklist

- [x] I've run the tests to see all new and existing tests pass
- [x] If this Pull Request resolves an issue, I linked to the issue in the text above
- [x] I've informed the technical writer about the documentation change if necessary

### CODE REVIEW

**Comment by whitewum** [comment]
This is a re-submission of #2142

File: README.md [hunk 1]
```diff
@@ -59,11 +58,11 @@ In case you encounter any problem, be sure to ask us on our [official forum](htt
 
 See our [Roadmap](https://github.com/vesoft-inc/nebula/wiki/Nebula-Graph-Roadmap) for what's coming soon in **Nebula Graph**.
 
-## Visualization Tool: Nebula Graph Studio
+## Graph Visualization
 
-Visit [Nebula Graph Studio](https://github.com/vesoft-inc/nebula-web-docker) for visual exploration of graph data on a web UI.
+Visit [Nebula Graph Studio](https://github.com/vesoft-inc/nebula-web-docker) for visual exploration of graph on browser.
```
Comment (Amber1990Zhang) [inline README.md:63]: Chrome browser?

Comment (whitewum) [reply README.md:63]: not necessary details at this page

File: README-CN.md [hunk 2]
```diff
@@ -41,9 +41,9 @@
 
 请查看[快速使用手册](https://docs.nebula-graph.com.cn/manual-CN/4.contributions/how-to-contribute/)，开始使用 **Nebula Graph**。
 
-在开始使用 **Nebula Graph** 之前，必须通过[编译源码](https://docs.nebula-graph.com.cn/manual-CN/3.build-develop-and-administration/1.build/1.build-source-code/)，[rpm/deb 包](https://docs.nebula-graph.com.cn/manual-CN/3.build-develop-and-administration/2.install/1.install-with-rpm-deb/) 或者 [docker compose](https://github.com/vesoft-inc/nebula-docker-compose/blob/master/README_zh-CN.md) 方式安装 **Nebula Graph**。您也可以观看[视频](https://space.bilibili.com/472621355)学习如何安装 **Nebula Graph**。
+通过[编译源码](https://docs.nebula-graph.com.cn/manual-CN/3.build-develop-and-administration/1.build/1.build-source-code/)，[rpm/deb 包](https://docs.nebula-graph.com.cn/manual-CN/3.build-develop-and-administration/2.install/1.install-with-rpm-deb/) 或者 [docker compose](https://github.com/vesoft-inc/nebula-docker-compose/blob/master/README_zh-CN.md) 方式安装 **Nebula Graph**。
 
-如果您遇到任何问题，请前往 Nebula Graph [官方论坛](https://discuss.nebula-graph.com.cn) 提问。
```
Comment (dangleptr) [inline README-CN.md:46]: Are you sure we remove the forum link in README?

**Pull request closed without merging**